Cargo.toml

```diff
@@ -109,3 +109,6 @@ required-features = ["random"]
 name = "test-predicate"
 path = "tests/predicate.rs"
 required-features = ["random"]
+
+[patch.crates-io]
+fuel-tx = { git = "https://github.com/FuelLabs/fuel-tx/" }
```

Comment:
we can remove this now right?

Reply:
We need to create a new release with `fuel-tx`. Do we want to create a new release for refactoring, and after a new release for `Mint`, or one release for both?

Reply:
I just approved the mint tx, so we can release both at once.